target/linux/bmips/dts/bcm6369-netgear-evg2000.dts

```diff
@@ -0,0 +1,242 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "bcm6368.dtsi"
+
+/ {
+	model = "Netgear EVG2000";
+	compatible = "netgear,evg2000", "brcm,bcm6369", "brcm,bcm6368";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_red;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 25 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 26 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led@2 {
+			label = "green:voip2";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+		};
+
+		led@4 {
+			label = "red:internet";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+		};
+
+		led@5 {
+			label = "green:internet";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+		};
+
+		led@14 {
+			label = "green:voip1";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+
+		led@15 {
+			label = "green:usb";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_green: led@22 {
+			label = "green:power";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		led_power_red: led@23 {
+			label = "red:power";
+			gpios = <&gpio 23 GPIO_ACTIVE_LOW>;
+		};
+
+		led@24 {
+			label = "green:lan";
+			gpios = <&gpio 24 GPIO_ACTIVE_LOW>;
+		};
+
+		led@26 {
+			label = "green:wifi";
+			gpios = <&gpio 26 GPIO_ACTIVE_LOW>;
```

Comment:
GPIO #26

Reply:
I think this one was copy&pasted from DGND3700v1 https://openwrt.org/toh/netgear/evg2000?s[]=evg2000#buttons but it's not documented in https://openwrt.org/toh/netgear/dgnd3700?s[]=evg2000#leds